src/lib/finishables.ts

```diff
@@ -195,6 +201,24 @@ export const finishables: Finishable[] = [
 		cl: moktangCL,
 		kill: () => MoktangLootTable.roll()
 	},
+	{
+		name: 'Kalphite King',
+		cl: kalphiteKingCL,
+		kill: ({ accumulatedLoot }) => {
+			const loot = new Bank();
+
+			const sortedDrops = [...drygoreWeapons].sort(
+				(a, b) => accumulatedLoot.amount(a) - accumulatedLoot.amount(b)
+			);
+			if (roll(drygoreDropChance)) {
+				const unowned = sortedDrops.find(d => !accumulatedLoot.has(d)) ?? sortedDrops[0];
+				loot.add(unowned);
+			}
+
+			loot.add(kalphiteKingLootTable.roll());
+			return loot;
+		}
+	},
```

Comment:
Couldnt the finishable/simulation directly call `KalphiteKingMonster.specialLoot()`? so that its more guaranteed its simulating using the real code.